index.js

```diff
@@ -418,43 +419,62 @@ function createPanZoom(domElement, options) {
     zoomByRatio(clientX, clientY, ratio);
   }
 
-  function centerOn(ui) {
-    var parent = ui.ownerSVGElement;
-    if (!parent)
-      throw new Error('ui element is required to be within the scene');
-
+  function centerOn(smooth) {
```

Comment:
The original idea of this method was for SVG elements, that would allow clients to center on a specific element of the scene.

As the library grew this become a bit obsolete in the context of WebGL/DOM scenes (which are also supported by the library).

May I propose to keep the API, but delegate the center inference to the controller? For example, [svgController API](https://github.com/anvaka/panzoom/blob/d19ab145814f32a91b1bd734f836497e022cd678/lib/svgController.js#L21) could have a method that takes the UI element and returns `cx/cy`, while [domController](https://github.com/anvaka/panzoom/blob/master/lib/domController.js) does the same for the DOM nodes?

I interpret this change as a desire to zoom into the center of the actual container, right?